sql/binlog_reader.h

```diff
@@ -270,6 +270,12 @@ class Basic_binlog_file_reader {
     Format_description_log_event *fd = read_fdle(offset);
     if (!fd) return has_fatal_error();
 
+    /* Invalid starting offset e.g. "SHOW BINLOG EVENTS FROM 5" */
```

Comment:
The check seems to be coupled with FDLE event position only: does it work with high invalid offsets?

Reply:
not in this commit